views/index.erb

```diff
@@ -74,6 +75,9 @@
               <br>
               <input type="checkbox" name="ps_remember" style="text-decoration: none !important;"></input>
             </div>
+            <div class="col-md-3 col-xs-5" style="margin-top: 1vh;">
+              <div style="color: #aeb1b7;">(Your account is not stored in our servers!)</div>
```

Comment:
This comment still needs to be resolved: https://github.com/menai/PowerGPA/pull/48#discussion_r86666639